test/modules/packages/distributedIters/perf/benchmarkDistributedIters.perfexecopts

```diff
@@ -0,0 +1,2 @@
+--test=uniform --mode=guided --n=10000 # benchmarkDistributeIters-guided.perfkeys
```

Comment:
Okay! Does this mean "use the default perfkeys file and store the result in distributedGuided.dat"? I could update the testing docs.